.github/workflows/test_python.yml

```
@@ -14,10 +14,9 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v3
       - run: sudo apt-get -q update
-      - run: sudo apt-get -qq upgrade
-      - run: sudo apt-get -qq install curl ffmpeg libcurl4-openssl-dev libssl-dev motion ssh v4l-utils
-      - run: pip install --upgrade pip wheel
-      - run: pip install build mypy pytest safety
+      - run: sudo DEBIAN_FRONTEND="noninteractive" apt-get -qq --no-install-recommends dist-upgrade
+      - run: sudo DEBIAN_FRONTEND="noninteractive" apt-get -qq --no-install-recommends install motion v4l-utils ffmpeg curl libcurl4-openssl-dev gcc libssl-dev
```

Avoid right-left scrolling, improve readability and sort dependancies to avoid duplicates.
```suggestion
      - run: sudo DEBIAN_FRONTEND="noninteractive" apt-get -qq --no-install-recommends install curl
                 gcc ffmpeg libcurl4-openssl-dev  libssl-dev motion v4l-utils
```